Tools/WebKitTestRunner/Configurations/WebKitTestRunner.xcconfig

```diff
@@ -43,11 +43,16 @@ EXCLUDED_SOURCE_FILE_NAMES_ = ;
 EXCLUDED_SOURCE_FILE_NAMES_cocoatouch = mac/*;
 INCLUDED_SOURCE_FILE_NAMES = mac/main.mm;
 
+MAC_CODE_SIGN_ENTITLEMENTS = $(MAC_CODE_SIGN_ENTITLEMENTS_$(USE_INTERNAL_SDK));
+MAC_CODE_SIGN_ENTITLEMENTS_ = Configurations/WebKitTestRunner.entitlements;
+MAC_CODE_SIGN_ENTITLEMENTS_YES = Configurations/WebKitTestRunner-internal.entitlements;
+
 CODE_SIGN_ENTITLEMENTS[sdk=macosx*] = Configurations/WebKitTestRunner.entitlements;
```

Comment:
`MAC_CODE_SIGN_ENTITLEMENTS` seems to remain unused.

Reply:
Good catch, thank you!